File: 백준/Bronze/2587. 대표값2/대표값2.cc
#include <iostream>
using namespace std;

int main() {
	int nArr[5];
	int sum = 0;
	for (int i = 0; i < 5; i++)
	{
		cin >> nArr[i];
		sum += nArr[i];
	}

	for (int i = 0; i < 4; i++)
	{
		for (int j = 0; j < 4; j++)
		{
			if (nArr[j] >= nArr[j + 1])
			{
				int tmp = nArr[j];
				nArr[j] = nArr[j + 1];
				nArr[j + 1] = tmp;
			}
		}
	}

	cout << sum / 5 << endl;
	cout << nArr[2] << endl;
}

File: 백준/Bronze/2587. 대표값2/대표값2.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int[] nArr = new int[5];
        int sum = 0;

        for (int i = 0; i < 5; i++) {
            nArr[i] = sc.nextInt();
            sum += nArr[i];
        }

        Arrays.sort(nArr);

        System.out.println(sum / 5);
        System.out.println(nArr[2]);
    }
}